hack/run-e2e-kind.sh

```diff
@@ -27,49 +27,58 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-export ROOT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/..
+export ROOT_DIR="$(dirname "$(dirname "$(readlink -fm "$0")")")"
 export LOG_LEVEL=3
-export CLEANUP_CLUSTER=${CLEANUP_CLUSTER:-1}
+export CLEANUP_CLUSTER=${CLEANUP_CLUSTER:-"true"}
 export CLUSTER_CONTEXT="--name test"
 # Using older image due to older version of kubernetes cluster"
-export IMAGE_NGINX="nginx:1.15.12"
-export IMAGE_ECHOSERVER="k8s.gcr.io/echoserver:1.4"
+export IMAGE_ECHOSERVER="kicbase/echo-server:1.0"
+export IMAGE_UBUNTULATEST="ubuntu:latest"
```

Comment:
should we use redhat ubi image?

Reply:
we could, but it would mean changing the e2e test to make use of it. Do we want to do this with this PR ?